src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,23 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
+        if (text.length() == 1 || text.length() == 0) {
+            return false;
+        }
+        text = text.toLowerCase();
+        text = text.replaceAll("[^A-Za-zА-Яа-я0-9]", "");
+        String[] arr = text.split("");
+        int count = 0;
+        for (int i = 0; i < arr.length; i++) {
```

Comment:
what about checklist rules? 
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_03.html

Reply:
Sorry for my old school I will change and try to be more attentive.